catt/subs_info.py

```diff
@@ -0,0 +1,51 @@
+import re
+
+import requests
+
+from .error import SubtitlesError
+from .util import create_temp_file
+
+
+class SubsInfo:
```

Comment:
We probably need docstrings here, It's not entirely clear at a glance what this class does.